lib/view_model/send/send_view_model.dart

```diff
@@ -760,4 +776,90 @@ abstract class SendViewModelBase extends WalletChangeListenerViewModel with Stor
 
     return false;
   }
+
+  EVMTransactionErrorFeesHandler parseEthereumFeesErrorMessage(
```

Comment:
**Enhancement**
make this a factory in the class itself, and move them to a separate file

Reply:
Alright